### PR TITLE
Neopixel Led Strip example added for ESP32

### PR DESCRIPTION
**Pull request by A13xs**
New samples for ESP32 RTM

## Description
Added two new solutions
* NeoPixelStrip
* NeoPixelStripLowMemory

## Motivation and Context
New example of using the RMT library

## Types of changes
- [ ] Improvement (non-breaking change that improves a sample)
- [ ] Bug fix (fixes an issue with a current sample)
- [x] New Sample (adds a new sample)

## Checklist:
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

Signed-off-by: A13xs

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/nanoframework/Samples?pullRequest=41) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/nanoframework/Samples?pullRequest=41) before we can accept your contribution.<br/><br/>:x: A13xs [sign now](https://cla.dotnetfoundation.org/nanoframework/Samples?pullRequest=41)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/nanoframework/Samples?pullRequest=41) it.</sub>

File: samples/Hardware.Esp32.Rmt/NeoPixelStrip/NeoPixel/app.config [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (josesimoes) [inline samples/Hardware.Esp32.Rmt/NeoPixelStrip/NeoPixel/app.config:1]: The app.config file shouldn't be here. It's added by VS on occasions and has to be deleted.

File: samples/Hardware.Esp32.Rmt/NeoPixelStripLowMemory/NeoPixel/app.config [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (josesimoes) [inline samples/Hardware.Esp32.Rmt/NeoPixelStripLowMemory/NeoPixel/app.config:1]: The app.config file shouldn't be here. It's added by VS on occasions and has to be deleted.

**Review by josesimoes — CHANGES_REQUESTED** [review]
Thanks for adding these.
Please address my review comments.
Also add an entry for these on the main README (both EN and CN versions please). 😁

**Comment by A13xs** [comment]
I removed the app.config

Global readme already has a link called "RTM interface" to the path Hardware.Esp32.Rmt

**Comment by josesimoes** [comment]
> Global readme already has a link called "RTM interface" to the path Hardware.Esp32.Rmt

You are correct, of course! I missed that in the rush...

**Review by josesimoes — APPROVED** [review]
Thanks for adding these! Any chance you can submit a project with this in Hackster.io? 😉

**Comment by josesimoes** [comment]
@A13xs are you on our Discord community? If yes, can you please ping me so I can properly credit your contribution?

**Pull request merged**